.eslintrc

```diff
@@ -22,12 +22,43 @@
     "no-undef": ["error"],
     "no-unused-vars": ["error", { "vars": "all", "args": "after-used" }],
     "no-use-before-define": ["error", { "functions": false, "classes": true }],
-    "no-var": ["error"],
     "quotes": ["error", "single"],
     "semi": ["error", "always"],
     "comma-dangle": ["error", "always-multiline"],
+    "guard-for-in": ["error"],
+    "valid-jsdoc": ["error",
+      {
+        "prefer": {
+          "return": "return"
+        },
+        "preferType": {
+          "String": "string",
+          "Number": "number",
+          "Boolean": "boolean",
+          "object": "Object",
+          "array": "Array"
+        },
+        "requireReturn": false,
+        "requireParamDescription": false,
+        "requireReturnDescription": false
```

Comment:
The JSDoc annotations checking is quite permissive at the moment. We'll make it stricter when we generate documentation from the code.